python/cudf/cudf/tests/test_string.py

```diff
@@ -1765,8 +1765,13 @@ def test_strings_filling_tests(data, width, fillchar):
     [
         ["A,,B", "1,,5", "3,00,0"],
         ["Linda van der Berg", "George Pitt-Rivers"],
-        ["+23", "³", "⅕", ""],
-        ["hello", "there", "world", "+1234", "-1234", None, "accént", ""],
+        ["³", "⅕", ""],
+        pytest.param(
+            ["hello", "there", "world", "+1234", "-1234", None, "accént", ""],
+            marks=pytest.mark.xfail(
```

Comment:
Once this PR is updated, @galipremsagar can you remove this in #11617?

Reply:
Yup, that's the plan. Waiting on this PR to be merged to un-xfail here: https://github.com/rapidsai/cudf/pull/11617/files#diff-abb1dc78dc17e099ea2d1f20a54e8b078edcf87e80865527b3a8e9660a66effcR1772